packages/firestore/test/integration/api/query.test.ts

```diff
@@ -1543,34 +1827,29 @@ apiDescribe('Queries', (persistence: boolean) => {
       });
     });
 
-    it('can use or queries with array membership', () => {
+    // eslint-disable-next-line no-restricted-properties
+    it('supports order by equality', () => {
       const testDocs = {
         doc1: { a: 1, b: [0] },
         doc2: { b: [1] },
-        doc3: { a: 3, b: [2, 7] },
+        doc3: { a: 3, b: [2, 7], c: 10 },
         doc4: { a: 1, b: [3, 7] },
         doc5: { a: 1 },
-        doc6: { a: 2 }
+        doc6: { a: 2, c: 20 }
```

Comment:
why has `c` been added to the test docs?

Reply:
It was not added. `it('supports order by equality'` was not modified. The diff is invalid.